istio-2-mixin/config.libsonnet

```diff
@@ -0,0 +1,33 @@
+{
+  // Static selector to apply to ALL dashboard variables of type query, panel queries, alerts and recording rules.
+  filteringSelector: 'job=~"integrations/istio"',
+  // Used to identify 'group' of instances.
+  groupLabels: ['job', 'cluster'],
+  instanceLabels: ['instance'],
```

Comment:
I remember you mentioning Istio has no such thing as "instances" the way we use them. It looks like this label only gets used one more time in the logs? Can we get rid of it?